ProjectTemplates/templates/Functions-CSharp/.template.config/dotnetcli.host.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
You probably want to copy most of the content of https://github.com/AzureAD/microsoft-identity-web/blob/master/ProjectTemplates/templates/WebApi-CSharp/.template.config/dotnetcli.host.json, except: 
- UseLocalDB
- OrgReadAccess
- DisableOpenAPI

Reply:
yes, forgot this one. thanks. done.

---
In reply to: [570252768](https://github.com/AzureAD/microsoft-identity-web/pull/903#discussion_r570252768) [](ancestors = 570252768)